examples/with-freactal/.babelrc

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
This can be removed. Since there is no special babel configuration

Reply:
@timneutkens You're right, this is uneeded, I removed it from the example.